cmd/pctl/main.go

```diff
@@ -282,6 +290,54 @@ func createPullRequest(c *cli.Context) error {
 	return catalog.CreatePullRequest(scmClient, g)
 }
 
+func prepareCmd() *cli.Command {
+	return &cli.Command{
+		Name:      "prepare",
+		Usage:     "prepare an environment with everything profiles need to work",
+		UsageText: "pctl prepare",
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:  "dry-run",
+				Usage: "If defined, nothing will be applied.",
+				Value: false,
+			},
+			&cli.BoolFlag{
+				Name:  "keep",
+				Usage: "Keep the downloaded manifest files.",
+				Value: false,
+			},
+			&cli.StringFlag{
+				Name:  "version",
+				Usage: "Define the tagged version to use which can be found under releases in the profiles repository. Exp: [v]0.0.1",
+			},
+			&cli.StringFlag{
+				Name:        "baseurl",
+				Usage:       "Define the url to go and fetch releases from.",
+				Value:       releaseUrl,
+				DefaultText: releaseUrl,
+			},
+			&cli.StringFlag{
+				Name:  "context",
+				Usage: "The Kubernetes context to use to apply the manifest files .",
+			},
+		},
+		Action: func(c *cli.Context) error {
+			p, err := cluster.NewPreparer(cluster.PrepConfig{
+				BaseURL:     c.String("baseurl"),
+				Version:     c.String("version"),
+				KubeConfig:  c.String("kubeconfig"),
+				KubeContext: c.String("context"),
+				DryRun:      c.Bool("dry-run"),
+				Keep:        c.Bool("keep"),
+			})
+			if err != nil {
+				return err
+			}
+			return p.Prepare()
+		},
+	}
+}
+
```

Comment:
can this be moved into a `prepare.go` in `cmd/pctl`? I've done that refactor for other commands in this PR, just to prevent one of us getting annoying merge conflicts :smile: https://github.com/weaveworks/pctl/pull/30

Reply:
Yes! I actually wanted to do that. :D I saw you rearranged things.